openvdb/INSTALL

```diff
@@ -64,6 +63,11 @@ Other versions might work but have not been tested.
 
 Installation
 ------------
+
+This documentation is for building with GNU Make, it is recommended to use
```

Comment:
Should this be worded more strongly, to say that Make support is deprecated and will be dropped in a future release?